integration/test/test_application_totals_pinning.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+"""Test that basic information expected when running the synthetic
+benchmark is reflected in the reports.
+
+"""
+
+import sys
+import unittest
+import os
+
+import geopmpy.io
+import geopmpy.agent
+import geopmdpy.error
+from integration.test import util
+from integration.test import geopm_test_launcher
+
+
+@unittest.skipUnless(geopm_test_launcher.detect_launcher() == "srun" or False,
```

Comment:
Please add documentation about why the test is being skipped in all cases due to the False statement that was added.